test/common.py

```diff
@@ -495,9 +498,9 @@ def accept_output(update_type):
                 return accept_output("output")
             else:
                 raise RuntimeError(
-                    ("I got this output for {}:\n\n{}\n\n"
+                    ("I got this output for {}{}:\n\n{}\n\n"
                      "No expect file exists; to accept the current output, run:\n"
-                     "python {} {} --accept").format(munged_id, s, __main__.__file__, munged_id))
+                     "python {} {} --accept").format(munged_id, subname_output, s, __main__.__file__, munged_id))
```

Comment:
This looks dodgy. Won't you get an incorrect command in this case?

Reply:
It's only on the error text, not the command. For example:

```
RuntimeError: I got this output for TestJit.test_test (my_subname):

my_subname

No expect file exists; to accept the current output, run:
python test/test_jit.py TestJit.test_test --accept
```

Reply:
right on.